src/constants/tooltip.js

```diff
@@ -0,0 +1,81 @@
+export const TOOLTIP_FORMAT_TYPES = {
```

Comment:
tooltip format should be linked with kepler field types,  you can add to default-settings `FIELD_OPTS` `format`

```js
  string: {
    type: 'categorical',
    scale: {
      ...ordinalFieldScaleFunctions,
      ...ordinalFieldAggrScaleFunctions
    },
    format: {
      legend: d => d,
      tooltip: [TOOLTIP_FORMAT_TYPES. decimal, TOOLTIP_FORMAT_TYPES. percentage]
    }
  },
```

Reply:
Okay